### DISCUSSION TITLE
Color lights do not show up in google home

### DISCUSSION THREAD

**Discussion by MHagreis** [andrei-tatar/node-red-contrib-smartnora#305] [Q&A]
when i add a switch it quickly Add's it to google home app
But when i add a color light it doesn't appear in google home app
for the switch i use the following node
![image](https://github.com/andrei-tatar/node-red-contrib-smartnora/assets/50264856/e095d78a-6a6d-4f27-9b86-548e4cae3a86)

for the color i user this node 
![image](https://github.com/andrei-tatar/node-red-contrib-smartnora/assets/50264856/b8c428e4-acca-4084-ab7d-a80c393172a1)

what am i doing wrong can some one give me some direction

**Comment by andrei-tatar** [comment]
@MHagreis Hi, did you sort this out?

**Comment by MHagreis** [comment]
Yes this is solved

Maarten Hagreis

________________________________
